src/main/java/io/github/sskorol/model/Parfume.java

```diff
@@ -8,10 +9,10 @@
  */
 @Data
 @EqualsAndHashCode(callSuper = true)
+@Source(source = "parfume.json")
 public class Parfume extends Product {
 
-    private String sortValue;
-    private String size;
+    private String amount;
```

Comment:
seems like a candidate for moving into more abstract level...
it might be also good to change its type...